filament/src/materials/skybox.mat

```diff
@@ -117,7 +117,7 @@ fragment {
     float3 shaprSkyGradient() {
         const float shaprThreshold = 0.5;
 
-        float upDirectionMapped = normalize(variable_eyeDirection).y * 0.5 + 0.5;
+        float upDirectionMapped = normalize(variable_eyeDirection).z * 0.5 + 0.5;
```

Comment:
So as discussed, I am fine with this as a quick solution, but an uniform variable would be similarly simple, and would enable using the same code in Shapr as on the artist branch